src/tests/components/ActivitiesListFilter.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ActivitiesListFilter from '../../components/ActivitiesListFilter';
+
+describe('ActivitiesListFilter', () => {
+  it('should render ActivitiesListFilter correctly', () => {
+    const wrapper = shallow(<ActivitiesListFilter />);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
snapshots are not super effiecient. I think, that at this moment you don't fully understand it. 
Try to figure better tests ;) You can try to find some elements there according to passed props, etc.